### DISCUSSION TITLE
Unwantedly Created Multiple accounts, Can I Either edit my mail or Have my one account deleted

### DISCUSSION THREAD

**Discussion by darkcheftar** [cssbattle/forum#299] [Q&A]
I unwantedly Created Two Profiles
1. [Two](https://cssbattle.dev/player/darkcheftar)
1. [One](https://cssbattle.dev/player/ubsH1oTrORazDf43Ff4uoZKgYSx1)
Want to Delete One.
Also would be happy if there is a way to change email

Thanks in advance!!!

**Comment by aarthvaen** [comment]
There's currently not possible to change the email associated with your account (hence it was suggested), but you should be able to reach out to devs via [contact@cssbattle.dev](mailto:contact@cssbattle.dev) and let them know you'd like one of the accounts to be deleted - providing the URL of the account you want deleted should be enough.

**Reply by darkcheftar** [reply]
Sure will Try that
